ilastik/applets/dataSelection/opDataSelection.py

```diff
@@ -204,6 +212,8 @@ def display_string(self) -> str:
 
     @property
     def default_output_dir(self) -> Path:
+        if self.project_file:
+            return Path(self.project_file.filename).parent
```

Comment:
maybe ensure an absolute path here (or even better when saving the attribute in the class)

Reply:
self.project_file keeps the actual h5py.File handle. I don't think we want to mess with that, so I left it at the readout point

Reply:
If you're happy with the `.absolute()` solution, I would go ahead and merge 🚀

Reply:
sure!